swat/cas/table.py

```diff
@@ -3506,7 +3505,7 @@ def _bool(self):
 
     def all(self, axis=None, bool_only=None, skipna=True, level=None, **kwargs):
         '''
-        Return whether all elements in the column are True
+        Return True for each column with only True elements
```

Comment:
This wording makes it sound like the items have to literally be True values, but it's really any value that is treated as true.  Maybe just not capitalize the second "True".

Reply:
Return True for each column with only elements treated as True